Base/usr/share/man/man7/KeyboardShortcuts.md

```diff
@@ -0,0 +1,65 @@
+
+# Keyboard Shortcuts
+
+This is a cheatsheet of common keyboard shortcuts in SerenityOS
+
+Related: [Tips and Tricks](help://man/7/Tips-and-Tricks)
+
+### Keywords
+* `Super` means the ⊞ Windows or ⌘ Command key
+* `Alt` is equivalent to ⌥ Option on a Mac
+  
+## General
+| Shortcut      | Action                                                    |
+|---------------|-----------------------------------------------------------|
+| `Super`       | Open the System Menu (navigate via arrow keys)            |
+| `Super+Space` | Summon Assistant (system search and application launcher) |
+
+## Window Management
+| Shortcut                 | Action                                                                |
+|--------------------------|-----------------------------------------------------------------------|
+| `Super+Up`               | Maximize the window                                                   |
+| `Super+Down`             | Minimize the window (or un-maximize)                                  |
+| `Super+Left`             | Snap window to left half of the screen                                |
+| `Super+Right`            | Snap window to right half of the screen                               |
+| `Super+LeftMouseButton`  | Move window (grabbing anywhere within)                                |
+| `Super+RightMouseButton` | Resize window (in the direction of mouse drag)                        |
+| `F11`                    | Fullscreen mode (not available in all applications)                   |
+| `Alt+Spacebar`           | Open window's context menu                                            |
+| `Super+D`                | Show and hide the Desktop                                             |
+| `Alt+Tab`                | Application Switcher (`Shift` reverses the order)                     |
+| `Super+Tab`              | Cycle focus between windows (`Shift` reverses the order)              |
+| `Super+Number`           | Switch to application in taskbar position 1-9 (or minimize if active) |
+| `Ctrl+Alt+Arrows`        | Move Workspace                                                        |
+| `Shift+Ctrl+Alt+Arrows`  | Move Workspace with the active window                                 |
+
+## Common within Applications
+| Shortcut       | Action                                    |
+|----------------|-------------------------------------------|
+| `Ctrl+Z`       | Undo                                      |
+| `Ctrl+Shift+Z` | Redo                                      |
+| `Ctrl+A`       | Select All                                |
+| `Ctrl+X`       | Cut                                       |
+| `Ctrl+C`       | Copy                                      |
+| `Ctrl+V`       | Paste                                     |
+| `Ctrl+O`       | Open File                                 |
+| `Ctrl+S`       | Save                                      |
+| `Ctrl+Shift+S` | Save As…                                  |
```

Comment:
No ellipsis needed in a description; in the UI they serve the specific function of "further input required after clicking this"